releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -84,6 +84,12 @@ If all of three filters are registered:
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeB, FilterB will be used and if context's type is TypeC, FilterC will be used.
 * When you call IsEnabledAsync("MyFeature", context), if context's type is TypeF, FilterA will be used.
 
+## 2.6.1 - June 28, 2023
+
+### Bug fix
+
+* Adds edge case for EvaluateAsync call that doesn't use context from FeatureManager.
```

Comment:
Fixed edge cases for ...

Is there an issue we can reference here?

Reply:
https://github.com/microsoft/FeatureManagement-Dotnet/issues/244

Reply:
Reference added.